### PR TITLE
Implement Service & Service Object for Compute

### PR DESCRIPTION
**Pull request by stephenplusplus**
For #904

This hooks Compute up with Service & ServiceObject.

Docs: http://stephenplusplus.github.io/gcloud-node/#/docs/master/compute

### CODE REVIEW

**Comment by stephenplusplus** [comment]
@callmehiphop rebased :)

**Comment by stephenplusplus** [comment]
Added the docs: http://stephenplusplus.github.io/gcloud-node/#/docs/master/compute

File: lib/compute/disk.js [hunk 1]
```diff
@@ -57,13 +70,92 @@ var util = require('../common/util.js');
...
-  this.zone = zone;
+  var methods = {
+    /**
+     * Create a persistent disk.
+     *
+     * @param {object} config - See {module:compute/zone#createDisk}.
+     *
+     * @example
+     * var config = {
+     *   // ...
```
Comment (callmehiphop) [inline lib/compute/disk.js]: Would a user have to go to the upstream docs to see the available configurations?

Comment (stephenplusplus) [inline lib/compute/disk.js]: Yeah, they go to `compute/zone#createDisk`, which is linked as the `config` parameter.

Comment (callmehiphop) [inline lib/compute/disk.js]: Ok cool

**Pull request merged**